### ISSUE TITLE
Use pypi-cli for getting data from PyPi

### ISSUE THREAD

**Issue by thinkingserious** [sendgrid/open-source-library-data-collector#3]
https://github.com/sloria/pypi-cli

**Comment by thinkingserious** [comment]
Now that download data is no longer on the web page, this is a bug

**Comment by jtemporal** [comment]
Hi @thinkingserious 😄 
I would like to take a shot at this.
Can you give me more info on what you have in mind here?

**Comment by thinkingserious** [comment]
Hello @jtemporal,

Thanks!

We used to scrape the [PyPi web page](https://github.com/sendgrid/open-source-library-data-collector/blob/master/package_managers.py#L58) for the package downloads, but that data is no longer available on the web side. As far as I know, you can only grab the data using the cli. So I would love to have the referenced web scraping function instead grab the data using the cli tool.

With Best Regards,

Elmer

**Comment by mikeclark** [comment]
Was just working on a similar project and noticed that pypi-cli was pulling in 0 dl counts for some big-name packages, so I thought I would mention this here:
https://stackoverflow.com/questions/38102317/why-pypi-doesnt-show-download-stats-anymore

There is a way to pull stats using the newest answer to the SO question above by using https://github.com/ofek/pypinfo but that requires a Google Cloud account and some setup on that side...

**Comment by thinkingserious** [comment]
@mikeclark,

Great find! Would you like to take that on as a [hacktoberfest](https://sendgrid.com/blog/announcing-hacktoberfest-2017/) PR?

With Best Regards,

Elmer

**Comment by mikeclark** [comment]
Yeah I can try to get this done, unless @jtemporal hasn't spent much time on it and your cool with using BigQuery

**Comment by jtemporal** [comment]
@mikeclark go ahead! I might take a while to get back to this sorry it took me so long to reply =)
